### ISSUE TITLE
Wrong package version in Ubuntu 16.04

### ISSUE THREAD

**Issue by rogermartensson** [antoineco/aco-tomcat#105]
The current version of the ubuntu 16.04 package for tomcat8 is 1.6.
Maybe it would be the best if not a specific version was set in the module. The parameter is still there for those who want to use a specific version.

**Comment by antoineco** [comment]
Yes this is a recurring issue with my design unfortunately. I did it that way because it is necessary to know the version in order to pull the correct `extra` libraries.

Instead of hardcoding these values I should have written a custom function that reads the available version from the package manager.
